cfgmgr/buffer_headroom_mellanox.lua

```diff
@@ -123,7 +128,7 @@ local speed_overhead
 
 -- Adjustment for 8-lane port
 if is_8lane ~= nil and is_8lane then
-    pipeline_latency = pipeline_latency * 2 - 1024
+    pipeline_latency = pipeline_latency * 2
```

Comment:
Why double the pipeline latency for 8lane?

Reply:
Hi
This is according to hardware implementation.